plan_script.sh

```
@@ -0,0 +1,2 @@
+sed -i 's/    @InjectMocks//g' src/test/java/com/sayai/record/fantasy/service/FantasyDraftServiceAutoPickTest.java
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)